.ci/eclipse-compiler-javac.sh

```diff
@@ -18,6 +18,8 @@ if [ ! -f $ECJ_PATH ]; then
     mkdir -p $(dirname "$ECJ_PATH")
     ECLIPSE_URL="http://ftp-stud.fht-esslingen.de/pub/Mirrors/eclipse/eclipse/downloads/drops4"
     wget $ECLIPSE_URL/$ECJ_MAVEN_VERSION/$ECJ_JAR -O $ECJ_PATH
+    echo "test jar after download:"
+    jar -tvf $ECJ_PATH > /dev/null
```

Comment:
We are directly storing the JAR in the cache path right away, right?
Won't `set -e` cause the `jar` command, and the entire CI entry, to fail on a corrupt jar and leave the jar in the cache?
Should we store the jar outside the cache and then move it in once this command passes?

Reply:
done.